packages/password-input/src/PasswordInput/PasswordInput.types.ts

```diff
@@ -43,7 +43,22 @@ interface StateOnlyProps {
   stateNotifications: State;
 }
 
-// using default notification container, no aria-describedby
+// Using general notification container, no aria-describedby
+interface GeneralNotificationProps {
```

Comment:
Can you add types tests for these in the spec file? We already have some type checks at the bottom of the file. I'm curious what TS will say if you pass a string to `stateNotifications`

Reply:
https://github.com/mongodb/leafygreen-ui/pull/2310/commits/16e2dcab8c5df02878f77f38faa96157026ba188